src/arrays/quarray.jl

```diff
@@ -34,6 +34,8 @@
     bases(qarr::QuArray, i) = qarr.bases[i]
     bases(qarr::AbstractQuArray) = ntuple(ndims(qarr), i->bases(qarr, i))
 
+    copy{B,T,N,A}(qa::QuArray{B,T,N,A}) = QuArray(copy(qa.coeffs), copy(qa.bases))
```

Comment:
It's not wrong to use the type parameters, but simply not necessary. `copy{B,T,N,A}=...` will give you a specialized version of `copy` for each combination of `B,T,N,A`. However, since `qa::QuArray` already matches all those types (the type without type parameters is an abstract type; see below for an example), you can omit the type parameters. Roughly speaking, if you don't need the parameters in your function or you don't want to constrain your types (for example in `*` we want the arrays to have the same bases, so we need `*{B}...`), you can most likely omit them.

Regarding the "parametrized types without parameters are abstract" point consider this:
```
type Foo{T<:Number}; n::T; end
julia> isleaftype(Foo)
false

julia> isleaftype(Foo{Int})
true

julia> Foo{Int} <: Foo
true

julia> Foo{Float64} <: Foo
true
```
It's sometimes a bit confusing, but reading the [manual section on parametric types](http://julia.readthedocs.org/en/release-0.3/manual/types/#man-parametric-types) 3-4 times helps :-)